File: src/main/java/com/subakstudio/wola/WoLa.java
package com.subakstudio.wola;

import com.subakstudio.wifi.WifiMacOSX;
import javafx.application.Application;
import javafx.fxml.FXMLLoader;
import javafx.scene.Parent;
import javafx.scene.Scene;
import javafx.stage.Stage;

import java.io.IOException;
import java.net.URL;

/**
 * Created by yeoupooh on 4/12/16.
 */
public class WoLa extends Application {

    public static void main(String[] args) {
        launch(args);
    }

    @Override
    public void start(Stage stage) {
        System.out.println(new WifiMacOSX().updateSignals());
        Parent root = null;
        try {
            URL fxml = getClass().getResource("/com/subakstudio/wola/WoLa.fxml");
            System.out.println(fxml);
            root = FXMLLoader.load(fxml);
            Scene scene = new Scene(root, 600, 400);
            stage.setTitle("WoLa");
            stage.setScene(scene);
            stage.show();
        } catch (IOException e) {
            e.printStackTrace();
        }
    }
}

File: src/main/kotlin/com/subakstudio/wola/WoLa.kt
package com.subakstudio.wola

import javafx.application.Application
import javafx.fxml.FXMLLoader
import javafx.scene.Parent
import javafx.scene.Scene
import javafx.stage.Stage
import java.io.IOException

/**
 * Created by Thomas on 4/12/2016.
 */
open class WoLa : Application() {
    companion object {
        fun main(vararg args: String) {
            launch(*args)
        }
    }

    override fun start(stage: Stage) {
        var root: Parent?
        try {
            val fxml = javaClass.getResource("/com/subakstudio/wola/WoLa.fxml")
            println(fxml)
            root = FXMLLoader.load<Parent>(fxml)
            val scene = Scene(root, 600.0, 400.0)
            stage.setTitle("WoLa")
            stage.setScene(scene)
            stage.show()
        } catch (e: IOException) {
            e.printStackTrace()
        }
    }
}